### PR TITLE
#interview

### PR DESCRIPTION
**Pull request by Noorhan-Khaled**

### CODE REVIEW

File: back-end/src/main/java/com/example/demo/controller/TopicLevelController.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import com.example.demo.model.TopicLevel;
+import com.example.demo.service.TopicLevelService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/topiclevel")
```
Comment (mhassanInfo90) [inline back-end/src/main/java/com/example/demo/controller/TopicLevelController.java:15]: change topiclevel to topic-level to make it more readable

File: back-end/src/main/java/com/example/demo/controller/TopicLevelController.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        topicLevelService.addTopicLevel(topicLevel);
+        return "Saved Successfully";
+    }
+    @PutMapping("/topiclevel")
+    public String updateTopicLevel(@RequestBody TopicLevelDto topicLevelDto){
+        TopicLevel topicLevel=topicLevelMapper.mapToEntity(topicLevelDto);
+        topicLevelService.updateTopicLevel(topicLevel);
+        return "Updated Successfully";
+    }
+    @DeleteMapping("/delete")
```
Comment (mhassanInfo90) [inline back-end/src/main/java/com/example/demo/controller/TopicLevelController.java:42]: also here

File: back-end/src/main/java/com/example/demo/dto/PositionDto.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package com.example.demo.dto;
+
+import javax.persistence.Column;
+
+public class PositionDto {
```
Comment (mhassanInfo90) [inline back-end/src/main/java/com/example/demo/dto/PositionDto.java:5]: you can use @Data Annotation and remove all getter and setter

Comment (Noorhan-Khaled) [reply back-end/src/main/java/com/example/demo/dto/PositionDto.java:5]: ok
الحاجات دى قديمة فرفعتها على طول

**Pull request merged**